data/json/monsters.json

```diff
@@ -4658,7 +4658,6 @@
     "luminance":0,
     "hp":20,
     "death_function":"MELT",
-    "special_attacks":[["VORTEX", 6]],
```

Comment:
I explained it in the PR description. It's a big, buggy (crashes the game), outdated 2D function only used for a single very rare monster. And now there's an excuse not to port it over.